cmake-edit-core/src/main/java/ru/urururu/cmakeedit/core/checker/SimulationState.java

```diff
@@ -51,73 +141,32 @@ public void accept(CommandInvocationNode node) {
             }
 
             private void processUsages(CommandInvocationNode node) {
-                if (node.getCommandName().equals("set")) {
-                    List<Node> arguments = node.getArguments();
+                List<Node> arguments = node.getArguments();
 
-                    if (arguments.isEmpty()) {
-                        return;
-                    }
-
-                    arguments.get(0).visitAll(new NodeVisitorAdapter() {
-                        @Override
-                        public void accept(ExpressionNode node) {
-                            processExpression(node, suspiciousPoints);
-                        }
-                    });
-
-                    for (int i = 1; i < arguments.size(); i++) {
-                        arguments.get(i).visitAll(new NodeVisitorAdapter() {
-                            @Override
-                            public void accept(ExpressionNode node) {
-                                processExpression(node, suspiciousPoints);
-                            }
-
-                            @Override
-                            public void accept(ArgumentNode node) {
-                                // not to hardcore?
-                                String argument = node.getArgument();
-                                processUsage(argument, suspiciousPoints);
-                            }
-                        });
-                    }
-
-                    return;
+                for (Node argument : arguments) {
+                    String arg = getNodeFromState(argument);
+                    getValue(arg); // todo shouldnt do for all!
                 }
-
-                node.visitAll(new NodeVisitorAdapter() {
-                    @Override
-                    public void accept(ExpressionNode node) {
-                        processExpression(node, suspiciousPoints);
-                    }
-
-                    @Override
-                    public void accept(ArgumentNode node) {
-                        // not to hardcore?
-                        String argument = node.getArgument();
-                        processUsage(argument, suspiciousPoints);
-                    }
-                });
             }
         });
     }
 
-    private void processExpression(ExpressionNode node, Set<Node> suspiciousPoints) {
+    private void processExpression(ExpressionNode node, Set<CommandInvocationNode> suspiciousPoints) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "processExpression" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)